From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 3 May 2014 11:27:34 +0200
Subject: Add missing import

---
 whatmaps/debiandistro.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/whatmaps/debiandistro.py b/whatmaps/debiandistro.py
index 54c706a..4a3d0db 100644
--- a/whatmaps/debiandistro.py
+++ b/whatmaps/debiandistro.py
@@ -26,6 +26,7 @@ except ImportError:
     lsb_release = None
 
 import logging
+import string
 import subprocess
 import sys
 
